src/main/java/org/students/simplebitcoinwallet/controller/TransactionsController.java

```diff
@@ -34,4 +32,16 @@ public TransactionsController(TransactionService transactionService) {
     public List<Transaction> getTransactions(@RequestParam String pubKey, @RequestParam String type) {
         return transactionService.getTransactions(pubKey, type);
     }
+
+    /**
+     * This method is used to create new transactions.
+     *
+     * @param transaction This is the transaction to be created.
+     * @return int This returns the result of the transaction creation.
+     */
+    @PostMapping("/send")
+    public int newTransactions(@RequestParam Transaction transaction){
```

Comment:
Please use @Valid annotation before declaring arguments. Also handle ValidationExceptions that might occur.